WordPress/Classes/ViewRelated/Plugins/PluginViewController.swift

```diff
@@ -167,3 +168,12 @@ private extension PluginViewController {
         return noResultsViewController
     }
 }
+
+extension PluginViewController: NetworkStatusDelegate {
+    func networkStatusDidChange(active: Bool) {
+        if active {
+            viewModel.networkStatusDidChange(active: active)
+            updateNoResults()
```

Comment:
This update seems to not be necessary. When the view model changes its state it triggers the no results update "automatically".